src/scenic/core/regions.py

```diff
@@ -1185,6 +1192,20 @@ def intersects(self, other, triedReversed=False):
                 return surface_collision
 
             # PASS 4
+            # If we have 2 candidate points and both regions have only one body,
+            # we can just check if either region contains the candidate point of the
+            # other. (This is because we previously ruled out surface intersections)
+            if (
+                s_candidate_point is not None
+                and o_candidate_point is not None
+                and self.mesh.body_count == 1
+                and other.mesh.body_count == 1
+            ):
+                return self.containsPoint(o_candidate_point) or other.containsPoint(
+                    s_candidate_point
```

Comment:
This is a pretty awful output of `black`. If you put parentheses around everything, can you get it to do something like
```python
return (
    self.containsPoint(o_candidate_point)
    or other.containsPoint(s_candidate_point)
)
```
instead?

Reply:
Unfortunately black won't let me change it :(